tests/cases/fourslash/findAllRefsThisKeyword.ts

```diff
@@ -25,7 +25,7 @@
 ////x.[|this|];
 
 const [global, f0, f1, g0, g1, x, y, constructor, method, propDef, propUse] = test.ranges();
-verify.singleReferenceGroup("this", [global]);
+verify.singleReferenceGroup("module globalThis\nthis: typeof globalThis", [global]);

```

Comment:
IIRC we wanted to suppress the `module globalThis` part of this to be less repetitive, at least. (Though I still think something like `(global) globalThis` would be nicer)

Reply:
I made the change in symbolDisplay. I'm not sure if it's right, although it's a filter on `!isThisExpression`, which is used in at least half of the other checks, so it's at least right/wrong in the same way as the rest of the code.